examples/viewer/viewer.cc

```diff
@@ -668,6 +775,13 @@ static void Init() {
   up[2] = 0.0f;
 }
 
+// Gopal: I am commenting out the original main function.
```

Comment:
Local change, not related to the fix. Please see my comments. I am refactoring the main function to suit the needs of my local project.